charts/argo-cd/templates/dex/deployment.yaml

```diff
@@ -76,14 +76,14 @@ spec:
         - name: grpc
           containerPort: {{ .Values.dex.containerPortGrpc }}
           protocol: TCP
```

Comment:
Can we restore this file? As far as I understand istio only cares about services, not Deployment/PodSpec

Reply:
I did it for you :) (https://github.com/argoproj/argo-helm/pull/1270/commits/7de74727d7137330eb642bbdb0185d10bbb142a5). I hope its okay